src/pages/Signup/Signup.js

```diff
@@ -40,6 +41,32 @@ function Signup() {
     setInputValues({ ...inputValues, [name]: value });
   };
 
+  const handleSignUp = () => {
+    const popType = isChecked ? 'nonPopup' : 'popupWrap';
+    setPop(popType);
+
+    fetch('http://10.58.52.134:8000/users/signup', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json;charset=utf-8' },
+      body: JSON.stringify({
+        name: inputValues.name,
+        email: inputValues.email,
+        phone_number: inputValues.number,
+        birthday: inputValues.date,
+        gender: inputValues.gender,
+        address: inputValues.address,
+        address_detail: inputValues.detail,
+        password: inputValues.password,
+        point: 30000,
+        agreement_private: inputValues.useage === 'on' ? 1 : 0,
+        agreement_marketing: inputValues.marketing === 'on' ? 1 : 0,
+        agreement_terms: inputValues.terms === 'on' ? 1 : 0,
```

Comment:
```suggestion
        agreement_private: inputValues.useage ? 1 : 0,
        agreement_marketing: inputValues.marketing  ? 1 : 0,
        agreement_terms: inputValues.terms  ? 1 : 0,
```
로 수정할 수 있습니다
empty string 의 boolean값은 false기 때문에  inputValues.useage  값이 있다면 true를 반환하겠죠?